### ISSUE TITLE
Set subject when sending files

### ISSUE THREAD

**Issue by guest4711** [grote/BlitzMail#28]
When you have shared a lot of files using thr wonderful Blitzmail it's hard to gind them in your mailfolder because alle subject header show the same (non-linformation: 1 file shared with Blitzmail.

So I would like to have an option in settings to allow Blitzmail to create a subject including the filename like [Blitzmail] name_of_transmittted_file

This would help a lot. I prefer an option for that because not everybody would like to have this info in metadata or server logs...

**Comment by ducalex** [comment]
I'd be happy to implement that. I have a few questions before I go ahead:

- Should it be able to display more than one file name, perhaps as many as can fit under 70 chars, eg:
   - `File1.jpg, File2.jpg, and 3 more files shared with Blitzmail` 
   - `File1.jpg and File2.jpg shared with Blitzmail`?
   - `File1.jpg shared with Blitzmail` 
- Is making it optional really necessary?

**Comment by grote** [comment]
Thanks for volunteering to pick this up!

The first option might still get problematic in other languages, but maybe it could cut whatever is too long?

I don't think there needs to be a setting for this.

**Comment by ducalex** [comment]
> The first option might still get problematic in other languages, but maybe it could cut whatever is too long?

Yes I was suggesting that BlitzMail would show as many filenames as can fit without exceeding a certain subject length. 

But maybe to avoid having to touch translations we could simply do:
- `File1.png | 1 File(s) shared with BlitzMail`
- `File1.png, ... | 8 File(s) shared with BlitzMail`

What do you think?

**Comment by grote** [comment]
Sounds good.  Maybe the suffix could even be shortened somehow. `.. | 8 files blitzMailed`.

Bonus point for using a plural string that uses the `s` only when needed.

**Comment by grote** [comment]
Or `File1.png, File2.jpg and 8 other files biltzMailed`.

**Comment by ducalex** [comment]
Sorry for the long delay! I'm back with some free time and the code is done we just need to decide on the format.

I'm fine with both your suggestions but they require a locale update for all languages, is that something we wish? (I don't know how much work it is).

**Comment by grote** [comment]
I personally don't mind that translations need to be adapted.

**Comment by ducalex** [comment]
After experimenting a bit with different formats I settled on something different. I've made a pull request with it but I really don't mind changing to any other format if you prefer, I just want this feature to be merged :).

Let me know if I need to update the translations (and how), it's only two words so I should be able to do all languages.
